docs/docs/workflows/tags.mdx

```diff
@@ -4,34 +4,42 @@ sidebar_position: 3
 
 # Creating Tags
 
-
-In Encord Active, you can tag your data to be able to [export](pages/export/filter_export) it.
+In Encord Active, you can tag your data to be able to [export](../pages/export/filter_export) it.
 Whether you are looking to relabel, augment, export, or delete data, the tagging feature is used in almost all Encord Active workflows.
 
-
- `Prerequisites:` Dataset (Labels are optional)
+`Prerequisites:` Dataset (Labels are optional)
 
 ### Steps:
+
 **Individual tagging:**
 How to tag individual images:
+
 1. Use the tagging feature from the _Data Quality_ and _Label Quality_ _Explorer_ tab.
 2. Access tags at the bottom of the left sidebar.
 3. Create tags of interest that you want group your data by (e.g., outliers, send to relabeling, delete)
 4. Tag images by clicking the dropdown menu located below the image (A single image can have multiple tags).
 5. View all tagged images in the _Actions_ tab by clicking tags within the _Add filter_ box.
 
-
 **Bulk tagging:**
 How to tag images in bulk:
+
 1. Use the tagging feature from the _Data Quality_ and _Label Quality_ _Explorer_ tab.
 2. Access tags at the bottom of the left sidebar.
 3. Create tags of interest that you want to group your data by (e.g., outliers, send to relabeling, delete). Consider the example of finding outliers.
 4. Use the bulk tagging menu bar to select _Outlier - Brightness_ and chose whether to perform bulk tagging on a page level or range level.
-  * If range level, chose the range using the sliders.
-  * If page level, all images on the current page will be tagged.
-6. Select _Add_ and _Submit_ bulk tagging (To undo tagging, select _Remove_ and resubmit).
-7. View all tagged images in the _Actions_ tab within the _Add filter_ box.
 
-<iframe width="800" height="450" src="https://www.loom.com/embed/ea9dd987e62244a78e1627a1438f3dee" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+- If range level, chose the range using the sliders.
+- If page level, all images on the current page will be tagged.
```

Comment:
Could there be an indent issue here?

Reply:
The indentation is apparently wrong with the original version as well
![image](https://user-images.githubusercontent.com/115489098/220340728-77d5d958-6941-42c2-bdc0-61c13441c5f5.png)

I will add a fix for proper indentation

Reply:
The indentation is apparently wrong with the original version as well
![image](https://user-images.githubusercontent.com/115489098/220340728-77d5d958-6941-42c2-bdc0-61c13441c5f5.png)

I will add a fix for proper indentation